### ISSUE TITLE
openwrt

### ISSUE THREAD

**Issue by suibaf** [babca/python-gsmmodem#79]
Hi,
do you think that python-gsmmodem work fine on a device with openwrt?
BR

**Comment by furryDev1lOwO** [comment]
Yes: I have tried it on Mikrotik router,flashed to latest openWRT firmware
